js/exercise-02.js

```diff
@@ -0,0 +1,14 @@
+/* 
+    1. Declare a variable using let with any name you like, but without a value.
+    2. On a separate line, assign the variable the String: "Moo!"
+    3. Log out this variable to the terminal using console.log
+    
+    Repeat the above steps but try using a const variable. Comment this new part of code and explain what is happening.
```

Comment:
There is not explanation for this case. The question is:
* did u try to execute this code initializing message as a `const` instead of `let`?
* could you successfully execute this script or did u have an error? Why?

FIX: try to execute this script using `const` instead of `let`. Show the output and explain the cause.